integration/experiment/sst_evaluation/gen_plot_time.py

```diff
@@ -0,0 +1,504 @@
+            dram_energy_to_solution = sum(data['Application Totals']['dram-energy (J)'] for host, data in report['Hosts'].items())
+
+            app_time_by_core = [time
+                                for host, data in report['Hosts'].items()
+                                for c,time in data['Application Totals'].items()
+                                if c.startswith('TIME@core-')]
+            app_network_time_by_core = [time
+                                for host, data in report['Hosts'].items()
+                                for c,time in data['Application Totals'].items()
+                                if c.startswith('TIME_HINT_NETWORK@core-')]
+            app_non_network_time_by_core = [time - net_time
+                for time, net_time
+                in zip(app_time_by_core, app_network_time_by_core)
+                if time > 0]
+
+            non_net_time_core_total = sum(app_non_network_time_by_core)
+            non_net_time_core_count = len(app_non_network_time_by_core)
+            non_net_time_core_max = max(app_non_network_time_by_core)
+
+            imbalance = (
+                (non_net_time_core_max - non_net_time_core_total / non_net_time_core_count)
+                * non_net_time_core_count
+                / (non_net_time_core_max * (non_net_time_core_count - 1))
+            )
+
+            app_name = report['Profile'][:report['Profile'].index('_' + report['Agent'])]
+            trial = int(report['Profile'].rsplit('_', 1)[-1])
+
+            runs.append({
+                'Profile': report['Profile'],
+                'Application': app_name,
+                'Agent': agent,
+                'Variant': variant,
+                'trial': trial,
+                'Time to Solution': time_to_solution,
+                'CPU Package Energy': package_energy_to_solution,
+                'DRAM Energy': dram_energy_to_solution,
+                'Energy to Solution': package_energy_to_solution + dram_energy_to_solution,
+                'Imbalance': imbalance,
+            })
+
+            for host, data in report['Hosts'].items():
+                trace_path = report_path.parent.joinpath(report_path.stem + f'.trace-{host}')
+                for field,value in data['Application Totals'].items():
+                    if field.startswith('CPU_FREQUENCY_STATUS@core-') and value != 0:
+                        core = int(field.split('@core-', 1)[-1])
+                        core_frequencies[(report['Profile'], core)] = {
+                            'Profile': report['Profile'],
+                            'Application': app_name,
+                            'Agent': agent,
+                            'Variant': variant,
+                            'trial': trial,
+                            'Frequency': value,
+                        }
+
+                sticker_frequency = (
```

Comment:
What does this represent?  You're taking the raw frequency in Hz over the percent of sticker frequency * 100... a comment would help.
Also unsure what the 1e8 * 1e8 is about.

Reply:
Adding a comment. This computes the sticker frequency from existing metrics for absolute mean frequency and relative mean frequency as a percentage of sticker in the report. The integer division followed by multiplication rounds down to a 100MHz step.